### PR TITLE
API-357: Add age verification helper method to HumanProfile

### PR DESCRIPTION
**Pull request by bucky-boy**
Here's the change to add isAgeVerified.
I've also taken this opportunity to update bouncy-castle to latest (1.59)

### CODE REVIEW

File: yoti-sdk-api/src/main/java/com/yoti/api/client/HumanProfile.java [hunk 1]
```diff
@@ -44,6 +44,13 @@
      */
     Date getDateOfBirth();
 
+    /**
+     * Did the user pass the age verification check?
+     *
+     * @return <code>TRUE</code> if they passed, <code>FALSE</code> if they failed, <code>null</code> if there was no check
+     */
+    Boolean isAgeVerified();
+
```
Comment (DarthSeppius) [inline yoti-sdk-api/src/main/java/com/yoti/api/client/HumanProfile.java]: only one question, we need to provide also the implemented one in the HumanProfileAdapter, right?

File: yoti-sdk-impl/src/main/java/com/yoti/api/client/spi/remote/HumanProfileAdapter.java [hunk 2]
```diff
@@ -87,6 +96,13 @@ public Date getDateOfBirth() {
         return wrapped.getAttribute(ATTRIBUTE_DOB, Date.class);
     }
 
+    @Override
+    public Boolean isAgeVerified() {
+        Boolean isAgeOver = wrapped.getAttribute(ATTRIBUTE_AGE_OVER, Boolean.class);
```
Comment (qzagarese) [inline yoti-sdk-impl/src/main/java/com/yoti/api/client/spi/remote/HumanProfileAdapter.java]: For what I can see, this cannot work.
The attribute starts with `age_over:` or `age_under:` (e.g. `age_over:18`)

I believe you need changes in the implementation of Profile where you can query for an `age_under:*` or `age_under:*` and return one if available.

Comment (DarthSeppius) [reply yoti-sdk-impl/src/main/java/com/yoti/api/client/spi/remote/HumanProfileAdapter.java]: Yep, see on this page `https://lampkicking.atlassian.net/wiki/spaces/IPASS/pages/29556930/Attribute+and+backing+document+names`

Comment (bucky-boy) [reply yoti-sdk-impl/src/main/java/com/yoti/api/client/spi/remote/HumanProfileAdapter.java]: Fixed

**Pull request merged**